src/main/java/edu/ie3/datamodel/models/input/container/SystemParticipants.java

```diff
@@ -270,4 +274,189 @@ public int hashCode() {
         storages,
         wecPlants);
   }
+
+  /**
+   * A builder pattern based approach to create copies of {@link SystemParticipants} containers with
+   * altered field values. For detailed field descriptions refer to java docs of {@link
+   * SystemParticipants}
+   *
+   * @version 3.1
+   * @since 14.02.23
```

Comment:
Why do we need this?

Reply:
The copy builder for entities, i.e ``UniqueEntitiyCopyBuilder`` also have this info in their class description. So I added them to the container copy builders.